packages/material-ui/src/internal/SwitchBase.test.js

```diff
@@ -1,28 +1,27 @@
 import * as React from 'react';
 import { expect } from 'chai';
 import { spy } from 'sinon';
-import { getClasses, createMount, describeConformance, act, createClientRender } from 'test/utils';
+import { createMount, describeConformanceV5, act, createClientRender } from 'test/utils';
 import SwitchBase from './SwitchBase';
 import FormControl, { useFormControl } from '../FormControl';
 import IconButton from '../IconButton';
+import classes from './switchBaseClasses';
 
 describe('<SwitchBase />', () => {
   const render = createClientRender();
   const mount = createMount();
-  let classes;
 
-  before(() => {
-    classes = getClasses(<SwitchBase icon="unchecked" checkedIcon="checked" type="checkbox" />);
-  });
-
-  describeConformance(
+  describeConformanceV5(
     <SwitchBase checkedIcon="checked" icon="unchecked" type="checkbox" />,
     () => ({
       classes,
       inheritComponent: IconButton,
       mount,
+      muiName: 'PrivateSwitchBase',
       refInstanceof: window.HTMLSpanElement,
       testComponentPropWith: 'div',
+      testVariantProps: { disabled: true },
+      skip: ['componentsProp', 'themeDefaultProps'],
```

Comment:
Let’s not skip the themeDefaultProps

Reply:
The test is returning `AssertionError: expected span.MuiButtonBase-root.MuiIconButton-root.PrivateSwitchBase-root.css-644pmx-MuiButtonBase-root-PrivateSwitchBase-root[aria-disabled="false"] to have an attribute 'id'`.  What should we do to resolve said error?

Reply:
We are testing to see if the `defalutProps` work with the `id` prop, but on this component it's being added on the `input` :). Let's choice some other `data-test` prop here.

```suggestion
       testThemeComponentsDefaultPropName: 'data-test',
      skip: ['componentsProp'],
```

In the future, please leave the failing test if there are any, so that reviewers can easily see the CI build failing and propose a solution.

Reply:
Gotcha -- okay, I'll keep that in mind going forward!